shared/cmd/flags.go

```diff
@@ -105,4 +105,10 @@ var (
 		Usage: "Specify log formatting. Supports: text, json, fluentd.",
 		Value: "text",
 	}
+	// MaxGoroutines specifies the maximum amount of goroutines tolerated, before a status check fails.
+	MaxGoroutines = cli.Int64Flag{
+		Name:  "max-healthy-goroutines",
```

Comment:
why not just `max-goroutines`? not sure if `healthy` is needed there

Reply:
+1

Reply:
```suggestion
		Name:  "max-goroutines",
```

Sorry, I was very tired when I suggested that name lol